### PR TITLE
switch migrations to use JS

### PR DESCRIPTION
**Pull request by dr-bonez**
requires embassy-sdk@0.3.1.1

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,5 +21,5 @@ image.tar: Dockerfile docker_entrypoint.sh check-rpc.sh configurator/target/aarc
 configurator/target/aarch64-unknown-linux-musl/btc-rpc-proxy: $(BTC_RPC_PROXY_SRC) $(CONFIGURATOR_SRC)
 	docker run --rm -v ~/.cargo/registry:/root/.cargo/registry -v "$(shell pwd)":/home/rust/src start9/rust-musl-cross:aarch64-musl sh -c "cd configurator && cargo +beta build --release"
 
-scripts/embassy.js: scripts/**/*.ts
+scripts/embassy.js: $(SCRIPTS_SRC)
```
Comment (Blu-J) [inline Makefile:24]: We may need to make this pattern in other places. I was oblivous to it not working before...

**Review by Blu-J — APPROVED** [review]
Proof:
Saw @dr-bonez going to and from versions to check that the migration system worked

File: manifest.yaml [hunk 2]
```diff
@@ -107,56 +107,10 @@ backup:
...
-      type: docker
-      image: main
-      system: false
-      entrypoint: "/bin/echo"
-      args: ['{"configured": true}']
-      io-format: json
-      inject: false
+    "*":
+      type: script
+      args: ["from"]
```
Comment (chrisguida) [inline manifest.yaml:112]: Where is this argument used?

**Review by chrisguida — APPROVED** [review]
LGTM!

**Review by islandbitcoin — APPROVED** [review]
ACK

**Pull request merged**